xsensmt/XsensMT.h

```diff
@@ -288,7 +339,8 @@ class yarp::dev::XsensMT : public yarp::dev::IGenericSensor,
     bool m_isSensorMeasurementAvailable{false};
 
     // Interface exposed by the Xsens MT Software suite
-    DeviceClass m_xsensDevice;
+	XsControl* m_xsensControl;
```

Comment:
```suggestion
    XsControl* m_xsensControl;
```

Reply:
Fixed in  https://github.com/robotology/yarp-device-xsensmt/pull/38/commits/0a495b087851691b7a989cbc3c4d0f030424db2b